awx/main/tasks.py

```diff
@@ -2071,16 +2100,20 @@ def build_passwords(self, inventory_update, **kwargs):
                 passwords[k] = credential.get_input(passkey, default='')
         return passwords
```

Comment:
build passwords for inventory updates doesn't make any sense to me. Does anyone know why this existed?